x-pack/solutions/observability/plugins/slo/server/saved_objects/slo.ts

```diff
@@ -49,13 +49,41 @@ export const slo: SavedObjectsType = {
         },
       ],
     },
+
+    2: {
+      changes: [
+        {
+          type: 'mappings_addition',
+          addedMappings: {
+            artifacts: {
+              properties: {
+                dashboards: {
+                  properties: {
+                    refId: { type: 'keyword' },
+                  },
+                },
+              },
+            },
```

Comment:
As mentioned by @rudolf in Slack:
We probably don't need these mappings, since we'll likely search by the `hasReference` API.